test/image-blueprints-bootc/layer2-source/group1/microshift-sshd-ownership.conf.template

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=/usr/bin/find /etc/ssh/ -name "ssh_*_key" -exec /bin/chgrp ssh_keys {} +
```

Comment:
I think it's a bit more subtle than this; we only want to change the group if it's not already owned by `root:root` because that's the new expected default in current Fedora and C10S.

Reply:
Possibly `/usr/bin/find /etc/ssh/ -name "ssh_*_key" -a '!' -gid 0 -exec /bin/chgrp ssh_keys {} +` or so

Reply:
While it'd have a large blast radius, maybe we should add this ssh one to the base image actually.

Reply:
I'd prefer not to do this in the base image because only specific source images are used in ostree-to-bootc upgrade procedures. The rest of the build still maintains the proper build hierarchy.

Reply:
Addressed part of the comments.